.upptimerc.yml

```diff
@@ -1,19 +1,19 @@
-owner: okp4
+owner: axone-protocol
 repo: status
 sites:
-  - name: OKP4 Web Site
-    url: "https://okp4.network"
-  - name: OKP4 Documentation Site
-    url: "https://docs.okp4.network"
-  - name: OKP4 Faucet
+  - name: Axone Web Site
+    url: "https://axone.xyz"
+  - name: Axone Documentation Site
+    url: "https://docs.axone.xyz"
+  - name: Axone Faucet
     url: "https://faucet.okp4.network"
-  - name: OKP4 Explorer
+  - name: Axone Explorer
```

Comment:
Thanks for pointing this out. However, regarding the Faucet, it hasn’t been migrated yet, so the URL should remain unchanged. We will consider this in a later step, but for now, since this repository is used to monitor our services, it’s crucial for us that it becomes operational quickly.